runtime/darwinia/Cargo.toml

```diff
@@ -316,6 +316,13 @@ try-runtime = [
 	"cumulus-pallet-xcmp-queue/try-runtime",
 	"parachain-info/try-runtime",
 
+	# darwinia-messages-substrate
+	"pallet-bridge-dispatch/try-runtime",
```

Comment:
This is done in #223 with some other extra changes. So please wait for that one merge.

Reply:
Please fix this I'm going to release Pangolin.